core.php

```diff
@@ -124,15 +124,8 @@ function require_lib( $p_library_name ) {
 		if( file_exists( $t_library_file_path ) ) {
 			require_once( $t_library_file_path );
 		} else {
-			global $g_vendor_path;
-			$t_library_file_path = $g_vendor_path . $p_library_name;
-
-			if( file_exists( $t_library_file_path ) ) {
-				require_once( $t_library_file_path );
-			} else {
-				echo 'External library \'' . $t_library_file_path . '\' not found.';
-				exit;
-			}
+			echo 'External library \'' . $t_library_file_path . '\' not found.';
```

Comment:
Is there a reason for not triggering an error here?

Reply:
It's been a while since I looked at that code, but IIRC this is because it is possible that not all of the API are loaded at that point.

Reply:
@vboctor you introduced it the way it is, see https://github.com/mantisbt/mantisbt/commit/b30a3124f26b4937932071add3c21e6396859485
I don't know if there was a reason to use echo+exit.
Any change concerning the error handling shouldn't be part of this PR.